app/Actions/Diagnostics/Pipes/Checks/ForeignKeyListInfo.php

```diff
@@ -40,12 +42,16 @@ private function sqlite(array &$data): void
 		$fks = DB::select("SELECT m.name , p.* FROM sqlite_master m JOIN pragma_foreign_key_list(m.name) p ON m.name != p.\"table\" WHERE m.type = 'table' ORDER BY m.name;");
 
 		foreach ($fks as $fk) {
-			$data[] = sprintf('Foreign key: %-30s → %-20s : %s', $fk->name . '.' . $fk->from, $fk->table . '.' . $fk->to, strval($fk->on_update));
+			$data[] = DiagnosticData::from(
```

Comment:
Why are these using `from` instead of `info`? Looks like the result would be the same?

Reply:
Good spot !